sbin/bectl/bectl.8

```diff
@@ -347,10 +347,20 @@ or
 .Fl a
 option is used.
 .It Cm mount Ar beName Op Ar mountpoint
-Temporarily mount the boot environment.
-Mount at the specified
+Temporarily mount the given boot environment.
+If a nonexistent
 .Ar mountpoint
-if provided.
+is given,
+.Nm
+will make the directory, including intermediate directories as required.
```

Comment:
"… intermediate directories as required." is from: 

<https://github.com/freebsd/freebsd-src/blob/d198b8774d2cfb6f140893e1c6236af9e97d1497/bin/mkdir/mkdir.1#L73>